ViAn/GUI/framewidget.cpp

```diff
@@ -189,7 +189,33 @@ void FrameWidget::set_anchor(QPoint p) {
 void FrameWidget::set_scale_factor(double scale_factor) {
     m_scale_factor = scale_factor;
 
-    if (m_vid_proj) m_vid_proj->get_video()->state.scale_factor = scale_factor;
+    if (m_vid_proj) {
+        m_vid_proj->get_video()->state.scale_factor = scale_factor;
+    }
+}
+
+/**
+ * @brief FrameWidget::set_rotation
+ * Slot function for updating framewidget after rotation have been done
+ * This will update the zoom rect coordinates
```

Comment:
Should be mentioned that it updates the current video project state as well
```suggestion
 * Updates the zoom rectangle coordinates as well as the rotation held by the current video project state
```